scripts/deploy.sh

```diff
@@ -217,7 +217,7 @@ function toolkit_refresh() {
     gitops_regcred_arg="--image-pull-secret ${secret_name}"
   fi
   mkdir -p ${work_dir}"/gitops"
-  gotk install --export --components=source-controller,helm-controller ${gitops_repo_arg} ${gitops_regcred_arg} > "${work_dir}"/gitops/gitops.yaml
+  flux install --export --components=source-controller,helm-controller ${gitops_repo_arg} ${gitops_regcred_arg} > "${work_dir}"/gitops/gitops.yaml
```

Comment:
this is a breaking change but we should all do it anyway

Reply:
Why is it breaking?

Reply:
you have to change your local cli from gotk to flux